framework/source/class/qx/test/Interface.js

```diff
@@ -71,8 +71,6 @@ qx.Class.define("qx.test.Interface",
         properties : { color : { } }
       });
 
-      var audi = new qx.test.i.Audi("audi");
```

Comment:
I don't know if we should delete this line. The instantiation itself could fail with an exception which then would maybe be a test itself.